### ISSUE TITLE
Inspector obscures selection on the left

### ISSUE THREAD

**Issue by 1ec5** [openstreetmap/iD#6917]
For #6461, the persistent sidebar was replaced by an assistant panel that shows only when there’s a selection, essentially returning iD to its original layout back in 2012, except with the inspector on the left instead of the right (in left-to-right environments).

The advantage to this layout is that it takes up less space when there’s no selection. Unfortunately, as noted in #109 and #265, if you select a feature off to the side, the assistant panel suddenly obscures it. It can seem like the selection is no longer being drawn on the map, and you have no idea what’s selected. In the screenshot below, I selected one of the motorways, which disappeared from view except at the margins:

![selection](https://user-images.githubusercontent.com/1231218/66260383-d2c62600-e772-11e9-8245-a0c727380b65.png)

The solution in #322 was to slowly nudge the viewport to the side to ensure the selection’s visibility. At the time, this approach was problematic both due to poor performance and visual instability when making rapid selection changes. Eventually, #1610 made the sidebar persistent, showing a preview of the inspector when hovering over a feature.

Another idea that was floated at the time was to have the inspector shift to the other side of the map if it would obscure the selection. I suspect iD didn’t go down this path because of its unpredictability.

**Comment by 1ec5** [comment]
Another way this issue plays out is that the “Zoom to fit” command zooms the map in such a way that the sidebar can obscure much of the selection, which isn’t what I’d normally want when using that command.

**Comment by quincylvania** [comment]
> Another way this issue plays out is that the “Center” command zooms the map in such a way that the sidebar can obscure much of the selection.

@1ec5 Yep, that's #6657. We should account for the top toolbar too, since vertical features can sometimes overlap it when centering.

**Comment by k-yle** [comment]
I'm going to close this issue, since it's a bug in the v3 prototype version, which is (sadly) abandoned and [no longer deployed](https://github.com/openstreetmap/iD/commit/36f4eeb7e24ba3ffd16f0d48780f3c56d39f2bd1).
